TestConsoleApp/BloggingContextFactory.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using TestConsoleApp.DbContext;
+
+namespace TestConsoleApp;
+
+public class BloggingContextFactory : IDesignTimeDbContextFactory<BloggingContext>
```

Comment:
Had to add this to get the migrations to run, otherwise it couldn't create the BloggingContext.